docs/pages/router/migrate/from-react-navigation.mdx

```diff
@@ -0,0 +1,468 @@
+---
+title: Migrate from React Navigation
+sidebar_title: from React Navigation
```

Comment:
```suggestion
sidebar_title: React Navigation
```

"from" looks weird in the sidebar:
<img width="284" alt="CleanShot 2023-07-10 at 18 42 22@2x" src="https://github.com/expo/expo/assets/10234615/b36649f2-83ee-46ad-aaea-57ae56f5c00e">

Let's keep it "React Navigation".

Reply:
Migrate "React Navigation" also doesn't seem right. Open to other suggestions.

Reply:
I think instead of having a section for "Migration" itself, we can re-title this page to: "Migrate from React Navigation" and move this doc inside the "Reference".

We follow this pattern for other sections as well. 

<img width="784" alt="CleanShot 2023-07-13 at 00 34 10@2x" src="https://github.com/expo/expo/assets/10234615/1a84630c-5788-49d1-9a8d-f04f70c05bcb">

The "from" doesn't follow the pattern of how we name doc titles in the sidebar. Alternatively, if you think "Migration" should be its own section, then let's change it to "From React Navigation" in the sidebar. Although, I'd push for the first suggestion since its the pattern that every section can follow (but at the same time I also understand since this is new section, there can be high traffic for folks and looking for migration guide -- but I don't have a concrete data to support that right now.